neurovault/apps/statmaps/migrations/0038_image_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django_hstore.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statmaps', '0037_auto_20150307_0222'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='image',
+            name='data',
+            field=django_hstore.fields.DictionaryField(null=True, blank=True),
+            preserve_default=True,
```

Comment:
API calls were failing for all images added before merging this PR. I had
to set 'data' to {} for all images.

On Thu, Aug 6, 2015 at 4:18 PM, Anton Burnashev <notifications@github.com>
wrote:

> In neurovault/apps/statmaps/migrations/0038_image_data.py
> <https://github.com/NeuroVault/NeuroVault/pull/309#discussion_r36477567>:
>
> > +from django.db import models, migrations
> > +import django_hstore.fields
> > +
> > +
> > +class Migration(migrations.Migration):
> > +
> > +    dependencies = [
> > +        ('statmaps', '0037_auto_20150307_0222'),
> > +    ]
> > +
> > +    operations = [
> > +        migrations.AddField(
> > +            model_name='image',
> > +            name='data',
> > +            field=django_hstore.fields.DictionaryField(null=True, blank=True),
> > +            preserve_default=True,
>
> I agree. How the problem manifested itself?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NeuroVault/NeuroVault/pull/309/files#r36477567>.
>